content/en/docs/spin-trigger-mqtt/contributing/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Contributing
+weight: 5
+description: >
+  Contributing to the Spin Trigger MQTT
+categories: [contributing]
+tags: [contributing, plugins, mqtt]
+---
+
+## Prerequisites
+
+To compile the plugin from source, you will need to:
+
+* Open the repository in Dev Container or in the pre-configured GitHub [Codespace](https://codespaces.new/spinkube/spin-trigger-mqtt).
```

Comment:
```suggestion
* Open the repository in a Dev Container or in the pre-configured GitHub [Codespace](https://codespaces.new/spinkube/spin-trigger-mqtt).
```

we should include non-devcontainer instructions for setup too - devcontainers are kinda inherently tied to vscode.